src/ec/packages/ec-component-expandable/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@ecl-twig/ec-component-expandable",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Expandable",
+  "dependencies": {
+    "@ecl-twig/ec-component-button": "^0.0.1-alpha"
+  },
+  "devDependencies": {
+    "@ecl/ec-component-expandable": "^2.1.1",
+    "@ecl/ec-specs-expandable": "^2.1.1",
+    "@ecl/ec-utility-typography": "^2.1.0"
```

Comment:
should be `^2.1.1` too

Reply:
I must have copied that wrongly from somewhere, fixed.